template/docker/Dockerfile.j2

```diff
@@ -30,9 +36,9 @@ RUN microdnf update -y --setopt=install_weak_deps=0 \
 COPY LICENSE /licenses/LICENSE
 
 COPY --from=builder /app/* /usr/local/bin/
-{[% if operator.include_productconfig is undefined or operator.include_productconfig == true %}]
-COPY deploy/config-spec/properties.yaml /etc/stackable/{[ operator.name }]/config-spec/properties.yaml
-{[% endif %}]
+# {[% if operator.include_productconfig is undefined or operator.include_productconfig == true %}]
```

Comment:
Are these `#` intentional?

Reply:
Yeah, they stop the linter failing the parse the Dockerfile.

Reply:
I also had to remove the spaces in `{[ operator.name }]` on the following line.

Reply:
Thank you